ui-v2/tests/acceptance/dc/nodes/index.feature

```diff
@@ -50,3 +50,29 @@ Feature: dc / nodes / index
     Then the url should be /dc-1/nodes
     Then I see 3 node models
     And I see leader on the healthyNodes
+  Scenario: Searching the nodes with name and IP address
+    Given 3 node models from yaml
+    ---
+      - Node: node-01
+        Address: 10.0.0.0
+      - Node: node-02
+        Address: 10.0.0.1
+      - Node: node-03
+        Address: 10.0.0.2
+    ---
+    When I visit the nodes page for yaml
+    ---
+      dc: dc-1
+    ---
```

Comment:
I'd possibly add a:

`And I see 3 node models`

in here, just to assert that we are starting off with 3 models before searching, what do you think?